src/components/ModalFlows/CreateProposalScreens/ActionFees.js

```diff
@@ -1,16 +1,16 @@
 import React, { useCallback } from 'react'
-import { Button, Field, GU, textStyle, theme } from '@1hive/1hive-ui'
+import { Button, Field, GU, textStyle, theme, useLayout } from '@1hive/1hive-ui'
 import iconFees from '../../../assets/iconFees.svg'
```

Comment:
We usually try to stick to a import order:
1- React
2- external libraries like 1hive ui
3- Providers and hooks
4- utils
5- assets